quinn-h3/src/tests/helpers.rs

```diff
@@ -47,6 +48,15 @@ pub struct Helper {
 
 impl Helper {
     pub fn new() -> Self {
+        if !TRACING_SUBSCRIBED.fetch_or(true, Ordering::Acquire) {
```

Comment:
A simpler approach would be to call [`tracing_subscriber::fmt::try_init`](https://docs.rs/tracing-subscriber/0.2.5/tracing_subscriber/fmt/fn.try_init.html) and ignore its return value.

Reply:
I was feeling this should be a thing, there it is. Thanks!